release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -339,6 +339,105 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-9-removed-features"]
 === Removed features
 
+==== Beta APIs removed from Kubernetes 1.22
+
+Kubernetes 1.22 removed the following deprecated `v1beta1` APIs. Migrate manifests and API clients to use the `v1` API version. For more information about migrating deprecated APIs, see the link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22[Kubernetes documentation].
+
+.`v1beta1` APIs removed from Kubernetes 1.22
+[cols=",,",options="header",]
+|===
+|Resource |API |Notable changes
+
+|`APIService`
+|`apiregistration.k8s.io/v1beta1`
+|No
+
+|`CertificateSigningRequest`
+|`certificates.k8s.io/v1beta1`
+|https://kubernetes.io/docs/reference/using-api/deprecation-guide/#certificatesigningrequest-v122[Yes]
```

Comment:
Also, even though it renders fine, all of these links should have `link:` in front of them. Like:

```
link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#certificatesigningrequest-v122[Yes]
```